grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -39,10 +43,12 @@ class WpsService extends AsyncDownloadService {
     }
 
     def notifyViaEmail(params) {
+
+        params.expirationPeriod = _getExpirationPeriod()
```

Comment:
If we make sure that there's always a value set in the config, and using joda, then I think this could be as simple as:

```groovy
params.expirationDateTime = new DateTime().plusSeconds(grailsApp.config.wpsResourceExpirationTimeout)
```

Then in the gsp:

```xml
... <prettytime:display date="${expirationDateTime}" />.
```

Not sure about the need for unit exclusion as in https://github.com/aodn/aodn-portal/pull/2023/files#diff-cf1774ac4be79e524ea06d4aaffd87efR72 - I would've thought this library would pick something sensible?